packages/mongodb-rag-core/src/MongoDbVerifiedAnswerStore.ts

```diff
@@ -0,0 +1,70 @@
+import { DatabaseConnection } from "./DatabaseConnection";
+import { VerifiedAnswer, VerifiedAnswerStore } from "./VerifiedAnswer";
+import { FindNearestNeighborsOptions, WithScore } from "./VectorStore";
+import {
+  MakeMongoDbDatabaseConnectionParams,
+  makeMongoDbDatabaseConnection,
+} from "./MongoDbDatabaseConnection";
+
+export function makeMongoDbVerifiedAnswerStore({
+  connectionUri,
+  databaseName,
+  collectionName,
+}: MakeMongoDbDatabaseConnectionParams & {
+  collectionName: string;
```

Comment:
```suggestion
  collectionName = "verified_answers",
}: MakeMongoDbDatabaseConnectionParams & {
  collectionName?: string;
```

could we do default value here?

Reply:
One problem with optional arguments: developers forget to pass the variable down and just use the default...